src/pages/library/common.js

```diff
@@ -427,7 +457,8 @@ const makeTable = (listData, sort, setSort, sortDir, setSortDir, selectedData, t
         ),
         div({ style: { ...styles.table.flexTableRow, alignItems: 'flex-start' } }, [
           div({ style: { ...styles.table.col, ...styles.table.firstElem } }, [
-            icon(listdatum.locked ? 'lock' : 'lock-o', { size: 12, style: { flex: 'none', color: listdatum.locked ? colors.accent() : colors.primary() } })
+            icon(listdatum.locked ? 'lock' : 'lock-o',
+              { size: 12, style: { flex: 'none', color: listdatum.locked ? colors.accent() : colors.primary() } })
```

Comment:
Thanks for reformatting this the way we normally do :)

```suggestion
            icon(listdatum.locked ? 'lock' : 'lock-o', {
              size: 12, style: { flex: 'none', color: listdatum.locked ? colors.accent() : colors.primary() }
            })
```